sql/executor.go

```diff
@@ -99,6 +99,11 @@ func (e *Executor) Execute(args driver.Request) (driver.Response, int, error) {
 		}
 		planMaker.setTxn(txn, planMaker.session.Txn.Timestamp.GoTime())
 	}
+	var err error
+	planMaker.evalCtx.Location, err = getTimeZone(planMaker.session)
```

Comment:
I'm going to move forward with pete's suggestion because the performance degredation is contained within the use of a timestamp. We'll fix any performance issues later.